api/views.py

```diff
@@ -120,3 +120,11 @@ class AssetTypeViewSet(ModelViewSet):
     permission_classes = [IsAuthenticated, ]
     authentication_classes = (FirebaseTokenAuthentication,)
     http_method_names = ['get', 'post']
+
+
+class AssetMakeViewSet(ModelViewSet):
+    serializer_class = AssetMakeSerializer
+    queryset = AssetMake.objects.all()
+    permission_classes = [IsAuthenticated, ]
+    authentication_classes = (FirebaseTokenAuthentication,)
+    http_method_names = ['get', 'post']
```

Comment:
could we do also return the asset make here as well instead of id? (for `get`). see comments [here](https://github.com/AndelaOSP/art-backend/pull/80/files)